include/arc_utilities/trajectory_utils.h

```diff
@@ -0,0 +1,8 @@
+#pragma once
```

Comment:
I disagree. `pragma once` is supported on the big 3 compilers, and is less error prone than include guards. I also don't think that's an important thing to have consistency with.

Reply:
It's not about compiler support.
I've looked into this multiple times, and there is some fundamental problem with some edge case of `#pragma once`. I think it has to do with the same file in multiple locations. IDEs are great at managing include guards.

Unfortunately this is one of those C++ cases where there are arguments on both sides. I wont deny the merge because of this. Feel free to resolve

Reply:
Yea I think this is what you mean: https://stackoverflow.com/a/47830826/3353601
I'm going to leave it like this, because if anyone is including multiple copies of `arc_utilities` header files they should have things explode in their face (they'll get a multiple-definition error and know something's up)